src/components/cart/proceed-payment/index.tsx

```diff
@@ -0,0 +1,206 @@
+            <RadioTitle>{data.offersRadio.title}</RadioTitle>
+            <RadioButton
+              idValue={data.offersRadio.idValue[0]}
+              radioName={data.offersRadio.name}
+              handleRadioClick={handleRadioClick}
+              option={data.offersRadio.options[0]}
+              checkedStatus={radioButtons.userCode}
+            />
+            {radioButtons.userCode === true ? (
+              <UserCodeSection>
+                <UserCodeInput
+                  name={data.offersRadio.userPromoCode}
+                  isPassword={false}
+                />
+                <AlertText content={data.offersRadio.alertsText} />
+              </UserCodeSection>
+            ) : null}
+            <RadioButtonOffers
+              idValue={data.offersRadio.idValue[1]}
+              radioName={data.offersRadio.name}
+              handleRadioClick={handleRadioClick}
+              option={data.offersRadio.options[1]}
+              checkedStatus={radioButtons.bestOffers}
+            />
+          </PaymentContainer>
+        </CartSection>
+      </ContentContainer>
+    </PageSection>
+  );
+};
+export default ProceedPayment;
+
+const PageSection = styled.div`
+  width: 100wh;
+  height: 842px;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  justify-content: space-between;
+  gap: 40px;
+  padding-bottom: 72px;
+`;
+
+const ContentContainer = styled.div`
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  justify-content: space-between;
+  gap: 39px;
+  margin-left: auto;
+  margin-right: auto;
+`;
+
+const ProgressSection = styled.div`
+  height: 55px;
+  width: 66%;
+`;
+
+const CookingInstructionTitle = styled.div`
+  height: 17px;
+  width: 122.95px;
+  color: ${colors.grey_858585};
+  font-family: ${fontFamilies.fontFamilyOsRegular};
+  font-size: ${sizes.size12};
+  letter-spacing: ${sizes.sizeNeg0_24};
+  line-height: ${sizes.size17};
+  margin-left: 21px;
+  margin-top: 17px;
+  margin-bottom: 15px;
+`;
+
+const CookingInstructionInput = styled(InputField)`
+  width: 263px;
+  margin: auto;
+  color: ${colors.grey_4a4a4a};
+`;
+
+const CartSection = styled.div`
+  width: 100%;
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  gap: 23px;
+  box-sizing: border-box;
+`;
+
+const StepsSection = styled.div`
+  width: 634px;
+`;
+
+const StepTitleLabel = styled(Label)`
+  height: "30px";
+  color: ${colors.black_000000};
+  fontfamily: ${fontFamilies.fontFamilyOsSemiBold};
+  fontsize: ${sizes.size22};
+  letterspacing: ${letterSpacing.space0};
+  lineheight: ${sizes.size30};
+`;
+
+const PaymentContainer = styled.div`
+  width: 305px;
+  height: 480px;
+  margin-top: 74px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 15px 17px 12px 18px;
+  border-radius: 6px;
+  box-shadow: 0 2px 10px 0 ${colors.black_000000_1};
+`;
+
+const RadioTitle = styled.div`
```

Comment:
No label here!!!!

Reply:
RadioTitle converted to RadioTitleLabel using Label styled component.

Reply:
Label common component used as styled tag.